src/sidebar.ts

```diff
@@ -144,6 +144,7 @@ export const reference = [
     ],
   },
   "reference/configuration",
```

Comment:
```suggestion
  "reference/node-configuration",
```

This is now needed, or you put both configurations on one page. I think this would work, since we have a TOC and are not limited by verticality.